### ISSUE TITLE
Making windows-session-counter more robust: Toast Notifications

### ISSUE THREAD

**Issue by meenubhooshanan** [IMLS/estimating-wifi#255]
### Description

(Depends on #239 being merged into main)

- [ ] Add a toast notification (or a pop up?) reprompting for inserting the USB wifi adapter
- [ ] Add a toast notification for a successful install
- [ ] Add a toast notification if there's an issue with the USB wifi adapter
